### ISSUE TITLE
accuracy problem with simple content

### ISSUE THREAD

**Issue by CanadianHusky** [tesseract-ocr/tesseract#3043]
### Environment

* **Tesseract Version**:  v5.0.0-alpha.20191030 with Leptonica
* **Platform**: Windows 7 and 10, 64bit

### Current Behavior:
Image in sample is detected wrong

![f024308c-2d89-4f9b-90d5-5a5f2a67916f](https://user-images.githubusercontent.com/35332003/85850372-5ae2d480-b7b5-11ea-8d77-b15fd7898677.png)

current result is **AlTITS**

command line cannot get any simpler. I also tried all reasonable PSM modes, 7 and 11 too
`>tesseract input.png output.txt --tessdata-dir "path to tessdata" --psm 3`

### Expected Behavior:
A1119 or AIII9 would be acceptable. 

### Suggested Fix:
1- Improve accuracy
2- Optionally, for short text of where result pattern is known in advance a command line option to specify somthing like 

```
? = any Letter or digit
* = letter only
# = digit only
  = space
. = any special char

```
or other regex based pattern validation to use as hinting for the OCR Engine, so it can decide more intelligently if A1119 or AIII9 makes more sense.

I tried the -c tessedit_char_whitelist= option but it is too restrictive and does not seem to work properly with special chars anyways, at least on the samples I have tested

I have lots of other example of a similar font with such mistakes and can supply them upon request
thank you kindly

**Comment by Shreeshrii** [comment]
There is a user patterns option but not sure if is working with tesseract4. 

If you know the font in  error, try to fine-tune for impact with just that font, 400 iterations and see if that helps.

See documentation for details.

**Comment by Shreeshrii** [comment]
As an experiment I finetuned with `Stylus` font and training text in all UPPERCASE for 800 iterations. I got a CER of 0.319 during lstmtraining and 0.412 with lstmeval with the eval files. The finetuned traineddata also recognizes the image correctly.

```
ubuntu@tesseract-ocr:~/tessdata_ocrb$ tesseract /home/ubuntu/TEST/A1119.png -
AlTITS
ubuntu@tesseract-ocr:~/tessdata_ocrb$ tesseract /home/ubuntu/TEST/A1119.png - -l stylus --tessdata-dir ./
A1119
ubuntu@tesseract-ocr:~/tessdata_ocrb$ tesseract /home/ubuntu/TEST/A1119.png - -l stylus_int --tessdata-dir ./
A1119
```

Zip file contains the `fast` version of traneddata.
[A1119.zip](https://github.com/tesseract-ocr/tesseract/files/5470334/A1119.zip)

**Comment by Shreeshrii** [comment]
See https://github.com/Shreeshrii/tessdata_stylus for details of training.

**Comment by CanadianHusky** [comment]
very helpful and good information, thank you
